CHANGES_UNRELEASED.md

```diff
@@ -1,6 +1,8 @@
 **See [the release process docs](docs/howtos/cut-a-new-release.md) for the steps to take when cutting a new release.**
 
-[Full Changelog](https://github.com/mozilla/application-services/compare/v91.0.0...main)
+# Unreleased Changes
```

Comment:
@bendk Re your comment about sneaking in a commit, this header is the only thing that seems anomalous so LGTM

Reply:
Oh yeah, I guess I should have left that header in, but it looks like the release script fixed things.